src/components/pages/order/OrderPage.js

```diff
@@ -6,12 +6,27 @@ import logoOrange from "../../../assets/logo-orange.png";
 import Menu from "./main/Menu";
 import { useState } from "react";
 import GlobalContext from "../../../context/GlobalContext";
+import { fakeMenu } from "../../../fakeData/fakeMenu";
 
 export default function OrderPage() {
   const { name } = useParams();
   const [isModeAdmin, setIsModeAdmin] = useState(false);
   const [panelTabIndex, setPanelTabIndex] = useState("add");
   const [isPannelCollapsed, setIsPannelCollapsed] = useState(false);
+  const [menus, setMenus] = useState(fakeMenu.LARGE);
```

Comment:
```suggestion
  const [menu, setMenu] = useState(fakeMenu.LARGE);
```

Pourquoi mettre un "s" à menu ? On a un seul menu pourtant. Si tu veux mettre un "s" pour indiquer qu'on parle d'un array, alors préférer [menuProducts, setMenuProducts]